### PR TITLE
Add changes to the pipeline configuration to deploy both the clusters

### PR DESCRIPTION
**Pull request by Ramparitala**
Made changes to deploy image to both live-1 and live clusters.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -143,6 +143,43 @@ jobs:
...
+      - run:
+          name: Authenticate to LIVE cluster
+          command: |
+            echo -n ${KUBE_ENV_LIVE_STAGING_CACERT} | base64 -d > ./ca.crt
+            kubectl config set-cluster ${KUBE_ENV_LIVE_STAGING_NAME} --certificate-authority=./ca.crt --server=https://${KUBE_ENV_LIVE_STAGING_NAME}
+            kubectl config set-credentials circleci --token=${KUBE_ENV_LIVE_STAGING_TOKEN}
+            kubectl config set-context ${KUBE_ENV_LIVE_STAGING_NAME} --cluster=${KUBE_ENV_LIVE_STAGING_NAME} --user=circleci --namespace=${KUBE_ENV_LIVE_STAGING_NAMESPACE}
+            kubectl config use-context ${KUBE_ENV_LIVE_STAGING_NAME}
+            kubectl config current-context
+            kubectl --namespace=${KUBE_ENV_LIVE_STAGING_NAMESPACE} get pods
```
Comment (zheileman) [inline .circleci/config.yml:163]: No need for these 2 lines, it is just debug from the cloud-platforms example.

File: .circleci/config.yml [hunk 2]
```diff
@@ -210,6 +238,12 @@ workflows:
           env_name: production
           requires:
             - tag_production
+      - deploy_image_to_live:
+          name: deploy-production-live
+          env_name: production
+          requires:
+            - tag_production
+
```
Comment (zheileman) [inline .circleci/config.yml:246]: Also no need for this workflow job just yet, as we don't have a production namespace on `live`.

**Review by zheileman — CHANGES_REQUESTED** [review]
Getting there. Just a few changes 👍

File: .circleci/config.yml (hunk 1 above)
Comment (Ramparitala) [reply .circleci/config.yml:163]: Noted, I will make changes and create a new fresh PR.

**Pull request closed without merging**